packages/certificate-model/package.json

```diff
@@ -19,43 +19,17 @@
     "access": "public"
   },
   "private": false,
-  "main": "dist/index",
-  "browser": "dist.browser/index",
-  "types": "dist/index",
-  "files": [
-    "dist"
-  ],
+  "main": "src/index.js",
+  "types": "src/index.d.ts",
   "engines": {
     "node": ">=12",
     "npm": ">=6"
   },
   "scripts": {
-    "build": "npm run clean && npm run compile",
     "clean": "rimraf -rf ./dist",
     "cleanup": "rimraf -rf dist coverage",
-    "compile:node": "tsc -p tsconfig.build.json",
-    "compile:browser": "tsc -p tsconfig.browser.json",
-    "compile": "npm run compile:node && npm run compile:browser",
-    "lint": "eslint -c ../../.eslintrc.js \"{src,apps,libs,test}/**/*.ts\" --fix",
     "prepublishOnly": "npm run build",
-    "test": "jest",
-    "test:cov": "rimraf -rf ./coverage && jest --ci --coverage",
-    "test:staged": "jest --runInBand --onlyChanged",
     "scan:deps": "audit-ci --config ../../audit-ci.json"
   },
-  "dependencies": {
-    "@s1seven/schema-tools-extract-emails": "^0.4.1",
-    "@s1seven/schema-tools-types": "^0.5.2",
-    "@s1seven/schema-tools-utils": "^0.2.5",
-    "@s1seven/schema-tools-validate": "^0.3.5",
-    "ajv": "8.12.0",
-    "ajv-formats": "2.1.1",
-    "lodash.clonedeepwith": "^4.5.0",
-    "lodash.merge": "^4.6.2"
-  },
-  "devDependencies": {
-    "@types/lodash.clonedeepwith": "^4.5.6",
-    "@types/lodash.merge": "^4.6.6"
-  },
   "gitHead": "5526910fd5974d09e80b5ba6324043b4409dd179"
```

Comment:
gitHead can be removed from all package.json,it was used when publishing through lerna.